### PR TITLE
fixes #14313. Instances of Chair/couch and its variances are now layered properly 

### LINKED ISSUE #14313: Chair/couch north overlays don't work until you rotate them
Built two chairs and only rotated the right one. Same issue with office chairs and couches at least.

untitled

### PR DESCRIPTION
**Pull request by Lancemaker**
Instances of Chair/couch and its variances are now layered (the variable is called Plane) properly. I added dir = 1 to the base object, so north will be the default chair direction.

### CODE REVIEW

**Comment by ghost** [comment]
noice

**Review by PJB3005 — CHANGES_REQUESTED** [review]
1. You shouldn't change the default direction. South (2) is always default. 
2. This is a really bad way to fix this in general, consider doing the following in `New()`:

```dm
if (ticker) 
    initialize()
```

**Comment by SonixApache** [comment]
add descriptive title, put [fixes x] in the description

**Comment by clusterfack** [comment]
The problem is that the construction dir isn't being passed as an argument to new, and its not being sent. Or rather that the construction code itself needs to call update_dir() on everything that gets constructed, and that this should have an update_dir() proc that calls handle_layer() (or just rename handle_layer() to update_dir())

**Comment by Lancemaker** [comment]
clusterfack,
what i understood was that there is an execution order to change the plane variable(that will in the end determine if the object is layered above or below the player). when the object is created using the recepy using iron stacks everything is pointing south, therefore the handle_layer() is useless to the created object independent to where the usr that created it is facing. There is another proc creating this object, and when it creates it changes its direction after the object is initialized. So my fix just changes the chair.dir to the usr.dir and adds the proper order and puts the handle_layer to work. 

PJB3005,

1. You shouldn't change the default direction. South (2) is always default. 
i will do that. 

2. This is a really bad way to fix this in general, consider doing the following in New():
``` 
if (ticker) 
    initialize()
```
But the New() is exactly like that : 

```
/obj/structure/bed/chair/New()
	..()
	if(ticker)
		initialize()
```

**Comment by SonixApache** [comment]
no you fool, make the title descriptive, like "properly layers couches"
and in the description (the first post) put "fixes x"
both without quotation marks

**Comment by Lancemaker** [comment]
SonixApache,
Sorry bro, im new to this, if you can point me to the formatting guidelines i will appreciate it.

**Comment by SonixApache** [comment]
This is better. don't think there's formatting guidelines defined anywhere, this is just what I think is better. I'm not sure if 'fixes X' is required to be in the OP to autoclose the issue if it's already in the commit messages, but you might as well throw it in there for good measure.

File: code/game/objects/structures/stool_bed_chair_nest/chairs.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/obj/structure/bed/chair
+/obj/structure/bed/chair
```
Comment (DamianX) [inline code/game/objects/structures/stool_bed_chair_nest/chairs.dm:1]: what's with all the extra spaces here and and there?

File: code/game/objects/structures/stool_bed_chair_nest/chairs.dm [hunk 2]
```diff
@@ -78,7 +79,7 @@
 
 	spin()
 
-/obj/structure/bed/chair/MouseDrop_T(mob/M as mob, mob/user as mob)
+/obj/structure/bed/chair/MouseDrop_T(mob/M as mob, mob/user as mob)//to bucle : if Mob has no butt, return.
```
Comment (DamianX) [inline code/game/objects/structures/stool_bed_chair_nest/chairs.dm:82]: buckle

**Comment by SonixApache** [comment]
Ded?

**Pull request closed without merging**